### DISCUSSION TITLE
help with sizing in inpaint

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2904] [Q&A]
So i have added a image in inpaint as a reception area that I want to add my model to

Masked the area that would be the right height of a person standing there

in " prompt " on the left when picking add objects,  i have added like __jessicaL__ standing u, wearing etc etc 

my images that come out look wrong and when i check the html log, there is nothing in the code to show it used the added information

> {
> "prompt": "Jessical is a English rose woman, blonde hair, ((blue eyes)), ((light freckles around nose and cheeks)), 32dd breasts, perfect eyes, perfect hands, standing in front of desk, wearing a pink crop top, white shorts, silver shoes, HDR",
> "negative_prompt": "(((ugly)))), (((duplicate))), ((morbid)), ((mutilated)), out of frame, extra fingers, mutated hands, ((poorly drawn hands)), ((poorly drawn face)), (((mutation))), (((deformed))), ((ugly)), blurry, ((bad anatomy)), (((bad proportions))), ((extra limbs)), cloned face, (((disfigured))), out of frame, ugly, extra limbs, (bad anatomy), gross proportions, (malformed limbs), ((missing arms)), ((missing legs)), (((extra arms))), (((extra legs))), (fused fingers), (too many fingers), (((long neck))), (burry), ((burry)), cropped, deformed, dull, poor lighting, deformed iris, deformed pupils, cropped, out of frame, jpeg,artifact",
> "prompt_expansion": "Jessical is a English rose woman, blonde hair, ((blue eyes)), ((light freckles around nose and cheeks)), 32dd breasts, perfect eyes, perfect hands, standing in front of desk, wearing a pink crop top, white shorts, silver shoes, HDR, highly detailed, dramatic light, excellent composition, dynamic, atmosphere, creative, ambient, lively, elegant",
> "styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
> "performance": "Speed",
> "resolution": "(768, 1216)",
> "guidance_scale": 4,
> "sharpness": 2,
> "adm_guidance": "(1.5, 0.8, 0.3)",
> "base_model": "juggernautXL_v8Rundiffusion.safetensors",
> "refiner_model": "None",
> "refiner_switch": 0.8,
> "sampler": "dpmpp_2m_sde_gpu",
> "scheduler": "karras",
> "seed": "5247814352146755744",
> "lora_combined_1": "JessicaL.safetensors : 1.0",
> "lora_combined_2": "PerfectEyesXL.safetensors : 1.0",
> "lora_combined_3": "Perfect Hands v2.safetensors : 1.0",
> "metadata_scheme": false,
> "version": "Fooocus v2.3.1"
> }

![2024-05-11_14-10-55_8994](https://github.com/lllyasviel/Fooocus/assets/35565479/eedf81e9-c648-4bbc-81c2-40edd5862202)

![2024-05-11_13-54-46_6664](https://github.com/lllyasviel/Fooocus/assets/35565479/541f9f5f-db24-47b5-89a8-0227f437aebb)

any ideals?

**Comment by lgwjames** [comment]
update

all added now  

![screenshot-127 0 0 1_7865-2024 05 12-09_47_13](https://github.com/lllyasviel/Fooocus/assets/35565479/3d2e7173-b392-44d7-8154-e253fb549d7d)

and a bit better, but has not put in my jessica girl from the lora?

![2024-05-12_09-34-16_4401](https://github.com/lllyasviel/Fooocus/assets/35565479/3f087459-492d-470a-9ee1-771feb9fa848)

**Comment by Rogala** [comment]
Enter your Promt in the main query field, select Inpaint or Outpaint (default) in the Inpaint or Outpaint mode and select both areas where to place the object.

I did in the mode lightning quality is not and Lora JessicaL.Safetensors I do not have. But basically everything works in both horizontal and vertical base of the picture.
![2024-05-13 170030](https://github.com/lllyasviel/Fooocus/assets/7437582/fff79315-83fb-4ec7-9296-b1cc3f310a98)
![2024-05-13 170105](https://github.com/lllyasviel/Fooocus/assets/7437582/73c6893d-19b9-481b-bb80-5478e954353c)
![2024-05-13_16-35-49_3631](https://github.com/lllyasviel/Fooocus/assets/7437582/1d63a07c-8372-43fa-84da-9a1c25f1b953)

**Comment by lgwjames** [comment]
thanks. will try when i get home
